### PR TITLE
fix issue adding booklist

### PR DESCRIPTION
**Pull request by SantoHerrera**

### CODE REVIEW

File: src/components/List.jsx [hunk 1]
```diff
@@ -31,6 +37,11 @@ const List = ({ id, list, booksInList, refetchLists, refetchBooks }) => {
     }
   }
 
+  function getListName(e) {
+    const elementClickedId = e.currentTarget.parentElement.parentElement.parentElement.parentElement.id;
+    setList(elementClickedId)
```
Comment (zorro1rr) [inline src/components/List.jsx:42]: I'm not crazy about this chain, we already have access to the id in the list prop

File: src/contexts/page-context.jsx [hunk 2]
```diff
@@ -19,6 +19,8 @@ export const PageProvider = ({ children }) => {
         setBooksItemCount,
         setListsItemCount,
         setPageNumber,
+        idOfListClicked,
+        setList
```
Comment (zorro1rr) [inline src/contexts/page-context.jsx:23]: this is crashing the app
image

**Comment by zorro1rr** [comment]
Instead of using context lets just pass the id as a query string.

File: src/contexts/page-context.jsx (hunk 2 above)
Comment (SantoHerrera) [reply src/contexts/page-context.jsx:23]: Yeah I think I forgot to check the line with this off
const [idOfListClicked, setList] = useState('1');

**Comment by SantoHerrera** [comment]
> 

Yeah that would probably be better

**Comment by SantoHerrera** [comment]
> Instead of using context lets just pass the id as a query string.

Yeah that would be better

**Pull request closed without merging**